arangod/IResearch/IResearchLinkCoordinator.cpp

```diff
@@ -83,6 +84,61 @@ IResearchLinkCoordinator::IResearchLinkCoordinator(
   _sparse = true;   // always sparse
 }
 
+Result IResearchLinkCoordinator::init(
+    velocypack::Slice definition,
+    IResearchLink::InitCallback const& initCallback) {
+  auto r = IResearchLink::init(definition, initCallback);
+  if (!r.ok()) {
+    return r;
+  }
+  using namespace metrics;
+  auto& metric = IResearchLink::collection()
+                     .vocbase()
+                     .server()
+                     .getFeature<ClusterMetricsFeature>();
+
+  auto batchToCoordinator = [](ClusterMetricsFeature::Metrics& metrics,
+                               std::string_view name, velocypack::Slice labels,
+                               velocypack::Slice value) {
+    ClusterMetricsFeature::MetricKey key{name, labels.stringView()};
+    std::get<uint64_t>(metrics[key]) += value.getNumber<uint64_t>();
```

Comment:
Just wondering: can there ever be any metrics with non-unsigned integer values, e.g. signed negative values, real values etc...? Or is that not possible by definition?
I know that at least in some parts of the code we have metrics that are floats internally and something like `metrics::Histogram<metrics::LogScale<float>>&`.

Reply:
This code works for uint64_t on the db server-side.
Also now ClusterMetrics don't works with histogram